### DISCUSSION TITLE
Mirage template not available

### DISCUSSION THREAD

**Discussion by zahiYoussef** [primefaces/community#1568] [PrimeNG Templates] [answered]
Hi Team , i wanted to bay Mirage templates for primeng 3 weeks ago , i loved it , now i  can't find it like it was removed .
are those templates getting back or they are removed forever ?

**Comment by rccueva** [comment]
I buy this template but not found this any more.

When will it be available again?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Mirage is an outdated template and will not receive an update anymore. However, we'll provide a manual guide to update templates to the newer versions.

**Reply by rccueva** [reply]
Hi,

My team is going to buy the Apollo template, they are asking when will this template be supported?

**Comment by shrestha-mausam** [comment]
I need to update Mirage 1.0.0 to work with Primefaces 11.0.0 ,
Currently, the icons for different components like tree, selectOneMenu, dialogs are not rendering correctly as expected 
Can you get me the guide to upgrade this template to work with Primefaces 11.0.0?
